sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/HigherOrderFunctionsSuite.scala

```diff
@@ -832,4 +832,10 @@ class HigherOrderFunctionsSuite extends SparkFunSuite with ExpressionEvalHelper
     assert(mapFilter2_1.semanticEquals(mapFilter2_2))
     assert(!mapFilter2_1.semanticEquals(mapFilter2_3))
   }
+
+  test("SPARK-36740: ArraySort add comment and unit test for Double.NaN and Float.NaN") {
```

Comment:
In this case, could you describe the content of this test case instead of PR title?

Reply:
Done, thanks for your suggestion.